src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
```

Comment:
This line a needless, because flex-direction: row; is a default value
<img width="571" alt="image" src="https://user-images.githubusercontent.com/86183007/185076329-11af84e5-9f15-4220-a885-eecf9b27fe57.png">